config.json

```diff
@@ -22,9 +22,23 @@
   },
   "exercises": {
     "concept": [],
-    "practice": []
+    "practice": [
+      {
+        "uuid": "d4919f7e-5e88-48a1-a357-e79cd6f98346",
+        "slug": "hello-world",
+        "name": "Hello World",
+        "practices": [
+          "basics"
+        ],
+        "prerequisites": [],
+        "difficulty": 1
+      }
+    ]
   },
   "concepts": [],
   "key_features": [],
-  "tags": []
+  "tags": [
+    "platform/windows",
+    "used_for/scripts"
+  ]
```

Comment:
I would try to add one (or more) of the `paradigm` and `typing` tags, as those are usually one of the first things students look for. `execution_mode` too. Runtime is probably `runtime/language_specific`?

I'm happy for this to be improved later on of course.